scss/mixins/_utilities.scss

```diff
@@ -41,22 +52,70 @@
       }
     }
 
+    $is-css-var: map-get($utility, css-var);
+    $is-local-vars: map-get($utility, local-vars);
     $is-rtl: map-get($utility, rtl);
 
     @if $value != null {
       @if $is-rtl == false {
         /* rtl:begin:remove */
       }
-      .#{$property-class + $infix + $property-class-modifier} {
-        @each $property in $properties {
-          #{$property}: $value if($enable-important-utilities, !important, null);
-          // Boosted mod: ensure contrasts in color utilities
-          @if "background-color" == $property and "transparent" != inspect($value) {
-            color: color-contrast($value);
-          } @else if "color" == $property and "inherit" != inspect($value) and $accessible-orange != $value {
-            background-color: color-contrast($value);
+
+      @if $is-css-var {
+        .#{$property-class + $infix + $property-class-modifier} {
+          --#{$variable-prefix}#{$property-class}: #{$value};
+        }
+
+        @each $pseudo in $state {
+          .#{$property-class + $infix + $property-class-modifier}-#{$pseudo}:#{$pseudo} {
+            --#{$variable-prefix}#{$property-class}: #{$value};
+          }
+        }
+      } @else {
+        .#{$property-class + $infix + $property-class-modifier} {
+          @each $property in $properties {
+            @if $is-local-vars {
+              @each $local-var, $value in $is-local-vars {
+                --#{$variable-prefix}#{$local-var}: #{$value};
+              }
+            }
+            #{$property}: $value if($enable-important-utilities, !important, null);
+
+            // Boosted mod: ensure contrasts in color utilities
```

Comment:
There must be some kind of regression here because pa11y shows us problems related to `text-muted` (e.g. https://deploy-preview-772--boosted.netlify.app/docs/5.0/examples/checkout/ vs https://boosted.orange.com/docs/5.0/examples/checkout/ where `text-muted` is displayed in white).

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/772/commits/3e1e3dab4819bfab9d225a620e85182668351a38